src/config.ts

```diff
@@ -500,6 +547,15 @@ function validateConfigAgainstV1(rawConfig: any, cliFlags: any) {
       'Whitelist & include specific folders via "mount" build scripts.',
     );
   }
+  // QUESTION: Should we warn about this being deprecated in the future?
+  if (
+    rawConfig.scripts &&
+    Object.keys(rawConfig.scripts).filter((k) => k.startsWith('proxy:`')).length > 0
+  ) {
+    handleDeprecatedConfigError(
```

Comment:
this will actually throw an error / exit, so it's a bit more than a "deprecation warning". I think it's fine to be silent about this for now.